src/main/java/org/minions/devfund/lourdes/MovieDatabase.java

```diff
@@ -0,0 +1,180 @@
+     * @return return the created movie.
+     */
+    public Movie createMovie(final String movieName) {
+        for (Movie movie : movieList) {
+            if (movie.getName().equals(movieName)) {
+                return movie;
+            }
+        }
+        Movie movie = new Movie(movieName);
+        movieList.add(movie);
+        return movie;
+    }
+
+    /**
+     * Method that create a new Actor object.
+     *
+     * @param actorName represents the actor name to be created.
+     * @return the created actor.
+     */
+    public Actor createActor(final String actorName) {
+        for (Actor actor : actorList) {
+            if (actor.getName().equals(actorName)) {
+                return actor;
+            }
+        }
+        Actor newActor = new Actor(actorName);
+        actorList.add(newActor);
+        return newActor;
+    }
+
+    /**
+     * Method that add a rating to a given movie.
+     *
+     * @param name   represents the movie name.
+     * @param rating rating of the movie.
+     */
+    public void addRating(final String name, final double rating) {
+        for (Movie movie : movieList) {
+            if (movie.getName().equals(name)) {
+                movie.setRating(rating);
+            }
+        }
+    }
+
+    /**
+     * Method that update the rating of the given movie name.
+     *
+     * @param name      represents the movie name.
+     * @param newRating the new rating to be set.
+     */
+    public void updateRating(final String name, double newRating) {
+        for (Movie movie : movieList) {
+            if (movie.getName().equals(name)) {
+                movie.setRating(newRating);
+            }
+        }
+    }
+
+    /**
+     * Method that return the best Actor.
+     *
+     * @return the best actor.
+     */
+    public String getBestActor() {
+        double highAverage = -1;
+        String bestActor = "";
+        for (Actor actor : actorList) {
+            if (actor.averageRating() > highAverage) {
+                highAverage = actor.averageRating();
+                bestActor = actor.getName();
+            }
+
+        }
+        return bestActor;
+    }
+
+    /**
+     * Method that return the best movie fo the list.
+     *
+     * @return the best movie.
+     */
+    public String getBestMovie() {
+        String bestMovie = "";
+        double bestRating = -1;
+        for (Movie movie : movieList) {
+            if (movie.getRating() > bestRating) {
+                bestRating = movie.getRating();
+                bestMovie = movie.getName();
+            }
+
+        }
+        return bestMovie;
+    }
+
+    /**
+     * Method that return the movie list.
+     *
+     * @return the movie list.
+     */
+
+    public ArrayList<Movie> getMovieList() {
+        return movieList;
+    }
+
+    /**
+     * Method that get the actor List.
+     *
+     * @return return the actor List.
+     */
+    public ArrayList<Actor> getActorList() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)